lib/util/include/hse_util/logging.h

```diff
@@ -101,14 +101,21 @@
     } while (0)
 
 
-#define log_info_sync(_fmt, ...) log_pri(HSE_LOGPRI_INFO, (_fmt), false, NULL, ##__VA_ARGS__)
-
+/* Main logging APIs for HSE.  Debug, info and warn levels use async
```

Comment:
Is this 80 column width? Hard to tell

Reply:
Wraps at 70.  Stupid emacs.  Gonna take me 4 days to convince emacs otherwise.

Reply:
It was set to 70, but even with fill=80, it wraps the same way.

Reply:
I just have an eye for this sort of thing :). This is why we need to get our clang format good to go, so we can just have our editors do this all for us automatically.